src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -40,9 +43,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
    return;
   }
   
-  String aboutMe = "This is " + user + "'s about me.";
+  User userData = datastore.getUser(user);
+
+  if(userData == null || userData.getAboutMe() == null){
+    return;
+  }
   
-  response.getOutputStream().println(aboutMe);
+  response.getOutputStream().println(userData.getAboutMe());
```

Comment:
Is there a way to populate this in the response without just printing to console?